README.md

```diff
@@ -1,27 +1,37 @@
-# Project Name
+# City Skyline
 
-> One paragraph statement about the project.
+> A city skyline by configuring CSS variables to be reused
 
 ![screenshot](./app_screenshot.png)
 
-Additional description of the project and its features.
-
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- ### HTML5 elements
+  - **Block element**
+    `div`
+- ### CSS properties
+  - CSS variables for reuseable CSS values,
+  - CSS media query for page responsiveness using `@media` rule,
+  - CSS positioning `absolute`,
+  - CSS flex-direction `column`
+  - CSS background for page aesthetic using `background-color`, `background-image`.
+  - CSS selectors `class selector` `pseudo selector` `universal selector` `root selector`
```

Comment:
You did not add the CSS selectors properties as you did here to the description of the project.